### PR TITLE
Event driven ansible

### PR DESCRIPTION
**Pull request by ashwinik30**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME | Removed existing package if any"
+fi
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    exit 0
```
Comment (nikhil-kalbande) [inline a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh:51]: This should be may be only exit.. Exit 0 means success state.

File: a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+cd $PACKAGE_NAME
+pip3 install -r test_requirements.txt
+pip3 install docker-compose build
+pip3 install awscli --ignore-installed six
+export ANSIBLE_HOME="/usr/local/bin"
+export PATH=$PATH:$ANSIBLE_HOME
+
+if ! ansible-galaxy collection install . ; then
+    echo "------------------$PACKAGE_NAME:test_fails---------------------------------------"
```
Comment (nikhil-kalbande) [inline a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh:62]: This message should be in install fails

File: a/ansible_event-driven-ansible/build_info.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "event-driven-ansible",
+    "github_url": "https://github.com/ansible/event-driven-ansible",
+    "version": "main",
+    "default_branch": "main",
+    "package_dir": "a/ansible_event-driven-ansible",
```
Comment (nikhil-kalbande) [inline a/ansible_event-driven-ansible/build_info.json:6]: Please use package folder like `ansible-event-driven-ansible`

File: a/ansible_event-driven-ansible/build_info.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "event-driven-ansible",
```
Comment (nikhil-kalbande) [inline a/ansible_event-driven-ansible/build_info.json:2]: here it should be ansible-event-driven-ansible

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh (hunk 1 above)
Comment (ashwinik30) [reply a/ansible_event-driven-ansible/event-driven-ansible_ubi_8_7.sh:51]: done

File: a/ansible_event-driven-ansible/build_info.json (hunk 3 above)
Comment (ashwinik30) [reply a/ansible_event-driven-ansible/build_info.json:6]: done changed as ansible_event-driven-ansible => **ansible-event-driven-ansible**

File: a/ansible_event-driven-ansible/build_info.json (hunk 4 above)
Comment (ashwinik30) [reply a/ansible_event-driven-ansible/build_info.json]: done

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**